integration/tests/posit/connect/test_content.py

```diff
@@ -10,6 +11,11 @@ def setup_class(cls):
             access_type="acl",
         )
 
+    @classmethod
+    def teardown_class(cls):
+        cls.item.delete()
+        assert cls.client.content.count() == 0
```

Comment:
I'm new to the architecture of this package — for my own learning, what does this do?

Reply:
`cls.item` is the content item created in the `setup_class` method. L16 performs a delete action, and L17 asserts that the total number of content items on the server is 0. The user used for testing has administrator privileges.

The general idea here is to try and ensure a "clean" server state between test classes. That way, other test classes would not need to be aware of each other. I'm not sure this is viable in the long term, but it's okay for now.